t/page/needed.rb

```diff
@@ -0,0 +1,39 @@
+require_relative '../../lib/page/needed'
+require 'minitest/autorun'
+require 'octokit'
+require 'pry'
+require 'vcr'
+
+describe 'Needed' do
+  VCR.configure do |config|
+    config.cassette_library_dir = 't/fixtures/vcr_cassettes'
+    config.hook_into :webmock
+    config.ignore_request do |req|
+      req[:uri].include? 'https://cdn.rawgit.com/everypolitician/everypolitician-data'
+    end
+  end
+
+  subject do
+    Page::Needed.new
+  end
+
+  it 'should return a list of countries labeled to_find' do
+    VCR.use_cassette('octokit - test to_find') do
+      list = subject.to_find
+      labels = ['New Country', 'To Find']
+      labels.must_include list.first[:labels].first[:name]
```

Comment:
I'm finding this test a bit too confusing to follow. As this is running against a canned response, perhaps it would be cleared to test something like `list.map(&:title).include? 'Eritrea'` ?